recipes/heliano/build.sh

```
@@ -0,0 +1,48 @@
+
+# set permissions to files
+cp ${PACKAGE_HOME}/heliano.py ${PACKAGE_HOME}/heliano
+cp ${PACKAGE_HOME}/heliano_cons.py ${PACKAGE_HOME}/heliano_cons
+chmod +x ${PACKAGE_HOME}/heliano
+chmod +x ${PACKAGE_HOME}/heliano_cons
+
+# test for conda
+df -h
+
+# put files in the executable bin
+cd ${PREFIX}/bin
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)